### ISSUE TITLE
Issue with tagwriter: In case of multiple genres per element a new TCON frame

### ISSUE THREAD

**Issue by Calliostro** [JamesHeinrich/getID3#224]
Hi James Heinrich

When saving multiple genres in ID3v2 format, a new TCON frame is generated for each element, which seems not to be allowed. MP3 Diags says "Multiple frame instances found, only the first one is used. (Frame: TCON)" (Example screenshot https://i.imgur.com/Smp7ou0.png).

When parsing genres you already consider the NULL byte separator. Maybe you could join several genres in this way when saving and write them in a single TCON frame?

Many greetings
Denis

**Comment by JamesHeinrich** [comment]
Yes, multiple genres are allowed in ID3v2, but they should all be written in a single TCON frame (although it's quite probable that getID3 does not currently handle it correctly).

Can you provide some simple example code of writing multiple genres that results in multiple TCON frames, to help me in testing and fixing the problem, please?

**Comment by Calliostro** [comment]
Thanks for the quick answer.

If you expand the genre list in **demos/demo.simple.write.php:45**:

`'genre' => array('Rock', 'Pop'),`

Then your tool **demos/demo.browse.php** shows a TCON array with 2 elements:

https://i.imgur.com/pKm9Xlv.png

MP3 Diags will display the warning.

Greetings
Denis

**Comment by Calliostro** [comment]
PS. When I write the IDv2 tags with the Mp3tag tool, the **demos/demo.browse.php** shows this as follows:

https://i.imgur.com/9YQfXsr.png
